frontend/src/components/EmbedSDK/Embed.js

```diff
@@ -1,27 +1,52 @@
-import React from 'react'
+import React, { useCallback }  from 'react'
 import styled from "styled-components"
 import { LookerEmbedSDK } from '@looker/embed-sdk'
 
-// create iframe of dashboard
-const DashboardDiv = (el) => {
-  LookerEmbedSDK.init(process.env.LOOKERSDK_EMBED_HOST, '/api/auth')
+const Embed = () => {
+  const makeDashboard = useCallback((el) => {
```

Comment:
Why use the hook useCallback when you aren't doing anything with dependencies? I think this is normally used when you need to wait on certain dependencies

Reply:
We have to control the lifecycle events, usecallback allows us to limit function to one time: usecallback(,[])
Otherwise as you browse the app it keeps appending additional iframes into the page